### ISSUE TITLE
is it possible to get from the api if the scheduler is enabled

### ISSUE THREAD

**Issue by yisraelU** [jhuckaby/Cronicle#339]

**Comment by mikeTWC1984** [comment]
Looks like documented api endpoints mainly related to event management, there is one for toggling state but I can't see one just for reading.
As a workaround you can capture cronicle state by hitting login api, it will send bunch of info in response, including state
**/api/user/login** 
There is another enpoint for basic server config which doesn't even require auth ( @jhuckaby   is it a bug or feature?)
**/api/app/config**
It does not contain information about state, but this could be a good place to add state info

**Comment by jhuckaby** [comment]
@mikeTWC1984 Yup, the `/api/app/config` is designed to be authless.  It kinda has to load first for the web UI to bootstrap itself.  It shouldn't contain any sensitive information (but also, I should say that Cronicle **should** run behind a firewall for security, but a lot of people don't).  I'll make sure this is redesigned for v2.0.
